amethyst_utils/src/ortho_camera.rs

```diff
@@ -159,7 +160,7 @@ mod test {
         let y = mat4.w.z; // c3r2
         let near = (y + 1.0) / x;
         let far = (x - 1.0) / y;
-
+    
```

Comment:
Yo, did rustfmt make a booboo?

Reply:
@azriel91 booboo? well, I can look into the book, in the case @jojolepro gets outvoted

I did it half manually, half automatic, but it only took 15 mins or so

Reply:
we'll see ;)

Reply:
booboo = there's 4 spaces, and I think rustfmt doesn't add trailing spaces if it can help it. It could've been a ctrl+tab losing control